### PR TITLE
enable al staging group capabilities

### PR DESCRIPTION
**Pull request by 4lejandrito**

### CODE REVIEW

**Comment by adolfopa** [comment]
Just started reviewing :)

File: portal-kernel/src/com/liferay/portal/kernel/util/GroupCapabilityProvider.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+package com.liferay.portal.kernel.util;
+
+import com.liferay.portal.kernel.model.Portlet;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface GroupCapabilityProvider {
+
+	public boolean supportsPages();
```
Comment (adolfopa) [inline portal-kernel/src/com/liferay/portal/kernel/util/GroupCapabilityProvider.java:24]: Shouldn't we rename this to "isPagesSupported()" for consitency? Same with the other method.

File: portal-kernel/src/com/liferay/portal/kernel/util/GroupCapabilityProvider.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+package com.liferay.portal.kernel.util;
+
+import com.liferay.portal.kernel.model.Portlet;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface GroupCapabilityProvider {
```
Comment (adolfopa) [inline portal-kernel/src/com/liferay/portal/kernel/util/GroupCapabilityProvider.java:22]: I find weird that we have a provider for something that we don't have (there's no GroupCapability object). Why not simply calling this GroupCapability?

**Comment by 4lejandrito** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/872.

**Pull request closed without merging**